package.json

```diff
@@ -3,13 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/free-solid-svg-icons": "^6.4.2",
```

Comment:
do we need both react icons and font awesome? could we try to work with only one? 
Please try to keep to one library.